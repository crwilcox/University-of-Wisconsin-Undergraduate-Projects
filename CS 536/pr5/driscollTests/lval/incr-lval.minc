void f(int s)
{
  ++s;
}
