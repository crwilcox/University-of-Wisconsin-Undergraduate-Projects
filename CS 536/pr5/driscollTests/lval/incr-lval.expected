void f(int s) {
  (++s); /* int */
}
